scanpipe/pipelines/load_sbom.py

```diff
@@ -21,15 +21,17 @@
 # Visit https://github.com/nexB/scancode.io for support and download.
 
 from scanpipe.pipelines.scan_codebase import ScanCodebase
-from scanpipe.pipes import scancode
+from scanpipe.pipes import resolve
 
 
-class ScanCodebasePackages(ScanCodebase):
+class LoadSBOM(ScanCodebase):
     """
-    Scan a codebase for PURLs without assembling full packages/dependencies.
+    Detects and loads all associated packages from SBOMs.
```

Comment:
We need to keep the same imperative style across: 

```suggestion
    Load package data from one or more SBOMs.
```